Automattic/jetpack
# Jetpack: fix visual issue when inserting patterns composed with videos

This PR fixes a visual issue that happens when inserting some patterns that are composed with VideoPress videos.

The patterns are formed with the`core/embed` VideoPress variation blocks, which define CSS classes (through block attributes). These classes break the VideoPress video block when the app converts them from embed to v6.

The fix is to clean these CSS classes when the embed block is detected before converting it to v6.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: fix visual issue when inserting patterns composed with videos

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






Test in simple sites
* Go to your testing site
* Install the VideoMaker theme
<img width="812" alt="Screen Shot 2022-12-09 at 14 07 46" src="https://user-images.githubusercontent.com/77539/206720368-539b2a8d-d347-4326-9674-6420e18b4fcd.png">

* Go to the block editor
* Open the block inserter
* Type `Header desc` to filter the pattern that we want to for testing
* 
<img width="370" alt="image" src="https://user-images.githubusercontent.com/77539/206720637-c7a191de-849e-4beb-8575-13db2afd5718.png">

* Confirm the pattern uses the VideoPress video block (it adds the `beta` label)
* Insert the pattern in the editor canvas
* Confirm it uses VideoPress video instances

before | after
-------|-------
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/77539/206723305-657c511b-8f83-4440-8dc6-ca82c68f7516.png"> | <img width="1122" alt="Screen Shot 2022-12-09 at 14 10 53" src="https://user-images.githubusercontent.com/77539/206720933-a46dd357-5473-4a37-b949-842fe133eef3.png">

* Confirm the height of videos are correct
